pages/index.tsx

```diff
@@ -26,57 +30,62 @@ const Home: NextPage<props> = ({ users }: props) => {
       return;
     }
 
-    setUsersData(response);
-    setNickname("");
-  };
-
-  const handleInput = (event: ChangeEvent<HTMLInputElement>): void => {
-    setNickname(event.target.value);
+    setUsersData(response.users);
+    setNickname(response.userNickname);
   };
 
   const renderJoinButton = () => {
     return (
       <Button
         value={nickname}
-        disabled={!nickname}
+        // TODO: disable button if input is empty
+        // disabled={!nickname}
         type="primary"
-        onClick={() => handleButtonClick(nickname)}
+        onClick={() => handleButtonClick()}
```

Comment:
Можно так, без создания доп функции
```suggestion
        onClick={handleButtonClick}
```

Reply:
done